update the README with pip installation instructions (#29)
prapti-ai/prapti: README.md

### 1. Install the `prapti` command line tool

> To make installation easier [Prapti is now on PyPI](https://pypi.org/project/prapti/).  

Reviewer: I don't think this is necessary.

```suggestion
```

> 

Reviewer: ```suggestion
```

In your terminal, run:

```
pip install prapti
```

(or `pip3` or `py -3 -m pip` depending on your system).

> To update to the latest version of Prapti through PyPI run:

Reviewer: ```suggestion
To update to the latest version of Prapti, run:
```

```
pip install prapti --upgrade
```

> If you wish to use the git repo directly you can do so by running:

Reviewer: ```suggestion
To install the the latest development version or Prapti directly from our git repo, run:
```

In your terminal, run:

```
pip install prapti
```

(or `pip3` or `py -3 -m pip` depending on your system).

To update to the latest version of Prapti, run:

```
pip install prapti --upgrade
```

> To install the the latest development version or Prapti directly from our git repo, run:

Reviewer: ```suggestion
To install the the latest development version of Prapti directly from our git repo, run:
```
